### ISSUE TITLE
[1.18, Bug] Server crash on startup

### ISSUE THREAD

**Issue by AlbiMousse** [GiantLuigi4/Smaller-Units#196]
-- Describe how you caused the bug --

Went to the server mod folder, (fabric 1.18.2), placed the mod, started up the server = Crash

-- Describe the expectation --

Self explanatory

-- Describe what happened instead --
Crash

-- Additional info --
SU Version: smallerunits-1.18.2-fabric-2.1.2
Exact MC Version: 1.18.2
Loader: fabric 1.18.2
Any optimization mods: N/A

-- Logs if the game crashed --
Even tho I called it a crash, I'm not sure about if it actually is. There isn't any thing in the log that ressembles a crash, and the only thing that shows is "[12:45:12] [Server thread/INFO]: Stopping server" after the servers start.
It does it even when the mod is alone with cardinal component and cloth api mods

-- If you’re using the CF launcher, feel free to attach an export of your modpack --
![image_2023-05-01_114657830](https://user-images.githubusercontent.com/116744793/235442428-a3d84306-3520-443d-9e25-16d3b73040bb.png)

**Comment by AlbiMousse** [comment]
Wait nvm I'm fucking blind here is a crash log that occured when I tried to start the mod
https://pastebin.com/DR08iMep

**Comment by LucidREM** [comment]
same server error. it does not play nice on servers??

**Comment by GiantLuigi4** [comment]
@LucidREM if you're also on 1.18, there is [this jar](https://cdn.discordapp.com/attachments/892136637604171806/1102700733872230510/smallerunits-1.18.2-fabric-2.1.2.jar)
If you're on 1.19, I'm not really sure, and I'm currently kindof on burnout from SU

**Comment by LucidREM** [comment]
> @LucidREM if you're also on 1.18, there is [this jar](https://cdn.discordapp.com/attachments/892136637604171806/1102700733872230510/smallerunits-1.18.2-fabric-2.1.2.jar) If you're on 1.19, I'm not really sure, and I'm currently kindof on burnout from SU

yeah it was 1.18.2 that was crashing. but if you are ever off burnout it would be amazing if 1.19.2 could work

**Comment by GiantLuigi4** [comment]
https://curseforge.com/minecraft/mc-mods/smaller-units/files/4627377 (1.18.2 fabric)
https://curseforge.com/minecraft/mc-mods/smaller-units/files/4627373 (1.19.2 fabric)
@LucidREM should work, amma go double check stuff
checked and yeah they work
